### PR TITLE
Refactored how we require shared middlewares from web/

### PR DESCRIPTION
**Pull request by kirrg001**
refs #9866

- use package notation
- get rid of x requires for middlewares
- improved readability
- do not refactor web/api/v0.1

### CODE REVIEW

**Comment by kirrg001** [comment]
@rishabhgrg Ready for review. Tests should pass 🤞

**Comment by kirrg001** [comment]
> Conflicting files
> core/server/web/admin/app.js

NOOOO 😁

File: core/server/web/site/app.js [hunk 1]
```diff
@@ -8,30 +8,10 @@ const apps = require('../../services/apps');
 const constants = require('../../lib/constants');
 const storage = require('../../adapters/storage');
 const urlService = require('../../services/url');
-
-// This should probably be an internal app
 const sitemapHandler = require('../../data/xml/sitemap/handler');
-
-// Route Service
+const themeService = require('../../services/themes');
```
Comment (rishabhgrg) [inline core/server/web/site/app.js]: Just wondering if this change should go in this PR, as its not related to shared middlewares from web/ and seems a bit odd one out. I get how this is a similar change, but then there will be more similar as well and then all of them can go as separate refactor maybe :)

Comment (kirrg001) [reply core/server/web/site/app.js]: Ok 👍 Will make 2 commits

Comment (kirrg001) [reply core/server/web/site/app.js]: I'll just undo.

**Pull request merged**